config/settings/development.yml

```diff
@@ -0,0 +1,8 @@
+host: <%= ENV['SETTINGS__SMTP__HOST'] %>
```

Comment:
@radinreth ideally, default environment should apply to Development and Test environment where Production shall override individually, please check the idea of the config/database.yml for your reference.